<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Search_Dialog</class>
 <widget class="QDialog" name="Search_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AiO's Search</string>
  </property>
  <widget class="QCheckBox" name="RegEx_CB">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>54</width>
     <height>17</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>RegEx</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SearchButton">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>250</y>
     <width>381</width>
     <height>23</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="Case_CB">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Case</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>361</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="Search_label">
      <property name="text">
       <string>Search Term:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="SearchEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QListWidget" name="listResults">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>381</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>280</y>
     <width>391</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Status Bar</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <zorder>RegEx_CB</zorder>
  <zorder>Case_CB</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>listResults</zorder>
  <zorder>label</zorder>
  <zorder>SearchButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
